pkg/diag/validator/pod.go

```diff
@@ -275,10 +276,12 @@ func extractErrorMessageFromWaitingContainerStatus(po *v1.Pod, c v1.ContainerSta
 
 func newPodStatus(n string, ns string, p string) *podStatus {
 	return &podStatus{
-		name:       n,
-		namespace:  ns,
-		phase:      p,
-		statusCode: proto.StatusCode_STATUSCHECK_SUCCESS,
+		name:      n,
+		namespace: ns,
+		phase:     p,
+		ae: &proto.ActionableErr{
```

Comment:
Does it need to be a pointer if it's always set?

Reply:
added this to be regular struct